--- a/src/nvim.c	2019-03-13 23:32:32.255831160 +0100
+++ b/src/nvim.c	2019-03-13 18:19:30.276502718 +0100
@@ -21,4 +21,5 @@
  */
 
+#include <assert.h>
 #include "eovim/nvim_api.h"
 #include "eovim/types.h"
--- a/src/nvim_request.c	2019-03-13 23:32:32.255831160 +0100
+++ b/src/nvim_request.c	2019-03-13 18:19:30.276502718 +0100
@@ -21,4 +21,5 @@
  */
 
+#include <assert.h>
 #include "eovim/nvim_api.h"
 #include "eovim/types.h"
